_episodes/06-workflow-management.md

```diff
@@ -323,7 +323,7 @@ rule NAME:
 ```
 - It is possible to address and offload to non-CPU resources:
 ```
-$ snakemake clean
```

Comment:
For my own interest, why is this?

Reply:
why was "snakemake clean" replaced by "snakemake --delete-all-output" ? It's considered better practice I believe. One can always forget to add things to the clean rule, while --delete-all-output will detect and delete all targets

Reply:
Ah, of course... `clean` is a rule, but `--delete-all-output` is a snakemake builtin that is auto-detected.